.travis.yml

```diff
@@ -88,7 +88,6 @@ jobs:
 cache:
   directories:
     - node_modules
-    - build-system/tasks/visual-diff/node_modules
```

Comment:
Am I right in understanding that the original problem was due to a bad cache on Travis that was missing certain modules, and that the cause was a bug in the way `yarn` was being run by `execOrDie` (that has since been fixed)?

With this change, we'll lose the advantages of caching, won't we? Would it not work to delete all existing caches (while continuing to enable caching for `visual-diff/node_modules` and `e2e/node_modules`), so that the Travis cache is freshly re-populated?

Reply:
> cause was a bug in the way yarn was being run 

Honestly not sure what caused the stale cache. I saw the `execOrDie` bug while I was debugging the chromedriver error but I don't know if that was the cause.

Ok by me to clear out the caches, seems like a nice thing to do routinely anyways. By not caching the visual-diff and e2e modules, we're talking about an additional 5s... :woman_shrugging: :woman_shrugging: :woman_shrugging: up to you!

Reply:
Just cleared out all old Travis caches. Hopefully this will fix the original issue.

I figured caching is still worth it because the `node_modules` directories in `visual-diff/` and `e2e/` add up to ~0.75 GB of downloads from npm servers per Travis build.

Reply:
Good point. Let me use this PR to re-include the e2e cache then.